### DISCUSSION TITLE
Broken link(s) in tutorial

### DISCUSSION THREAD

**Discussion by maddenp-cu** [dtcenter/METplus#2732] [Training] [answered]
The [Timing Control in METplus](https://dtcenter.org/metplus-practical-session-guide-version-5-0/session-1-metplus-setupgrid-grid/metplus-overview/modifying-timing-control-example-conf) page has a link in the first paragraph with text "Timing Contol" and target https://metplus.readthedocs.io/en/latest/Users_Guide/configuration.html#timing-control. The target is invalid.

If I find others while working through the tutorial, I'll follow up here.

**Comment by jprestop** [comment, marked as answer]
@maddenp-noaa Thanks so much for bringing that to our attention.  I just fixed the link.  Hopefully, if you refresh the page, it will work for you now.

**Comment by maddenp-cu** [comment]
Looks good now, thanks!

**Reply by jprestop** [reply]
Thanks for letting us know!
